### PR TITLE
Drupal parsing

### PR DESCRIPTION
**Pull request by jacobaaronyeager**

### CODE REVIEW

File: project_conf/scripts/parse_drupal_keys.lua [hunk 1]
```diff
@@ -0,0 +1,29 @@
+-- gets drupal record and returns a record with a json string of the drupal attributes
+function parse_keys_with_eq_pairs(tag, timestamp, record)
+    print("record: ")
```
Comment (akf) [inline project_conf/scripts/parse_drupal_keys.lua:3]: Remove print statements

File: project_conf/scripts/parse_http_headers.lua [hunk 2]
```diff
@@ -0,0 +1,14 @@
+-- Convert the http_headers field (block of text) to a table of key:value pairs
```
Comment (akf) [inline project_conf/scripts/parse_http_headers.lua:1]: Remove this file (duplicate of the one in cg-logshipper/scripts)

File: project_conf/scripts/parse_keys_with_eq_pairs.lua [hunk 3]
```diff
@@ -0,0 +1,26 @@
+-- This parser is just a proof-of-concept.
```
Comment (akf) [inline project_conf/scripts/parse_keys_with_eq_pairs.lua:1]: Remove this file (duplicate of the one in cg-logshipper/scripts, but with print statements).

**Review by akf — CHANGES_REQUESTED** [review]
I noted a couple of files to delete and a function I'd like to see renamed.

**Comment by jacobaaronyeager** [comment]
@akf Updated

**Review by akf — APPROVED** [review]

**Pull request merged**